### PR TITLE
RNs-4.9.51 Added with Bug Fix & Security Update

### PR DESCRIPTION
**Pull request by Wiharris**
**RNs-4.9.51**
Added with Bug Fix & Security Update

**Version(s):**
* PR applies only to enterprise-4.9

**Issue:**
https://issues.redhat.com/browse/OSDOCS-4434

**Link to docs preview:**
https://wiharris.github.io/openshift-docs/RNs-4.9.51/release_notes/ocp-4-9-release-notes.html#ocp-4-9-51

QE review:
- [ ] QE has approved this change.

**Additional information:**
`Shipped Live`

### CODE REVIEW

**Review by jldohmann — COMMENTED** [review]
Mostly LGTM! Left a question about some links below

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -3268,4 +3268,30 @@ $ oc adm release info 4.9.50 --pullspecs
...
-To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
\ No newline at end of file
+To update an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-9-51"]
+=== RHSA-2022:7216 - {product-title} 4.9.51 bug fix and security update
+
+Issued: 2022-11-02
+
+{product-title} release 4.9.51, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:7216[RHSA-2022:7216] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:7215[RHBA-2022:7215] advisory.
```
[the remark this answers is not in the document]
Comment (Wiharris) [reply release_notes/ocp-4-9-release-notes.adoc:3278]: That is correct! The links will not show until the erratas are officially `shipped live` :)

[the remark this answers is not in the document]
Comment (jldohmann) [reply release_notes/ocp-4-9-release-notes.adoc:3278]: Awesome! Thanks for clarifying!

**Review by bergerhoffer — COMMENTED** [review]
LGTM for merging!

**Pull request merged**